test/specs/promise.spec.js

```diff
@@ -67,4 +69,22 @@ describe('promise', function () {
       done();
     }, 100);
   });
+
+  it('should allow for configuring Promise constructor', function(done) {
+    var defaultConfig = {
+      Promise: Bluebird
+    };
+    spyOn(defaultConfig, 'Promise');
```

Comment:
You don't actually need `Bluebird`. You can just create a spy function from scratch:

```javascript
var defaultConfig = jasmine.createSpyObj('defaultConfig' ['Promise']);

...

expect(defaultConfig.Promise).toHaveBeenCalled();
```